plugins/woocommerce/src/Admin/Marketing/InstalledExtensions.php

```diff
@@ -319,6 +377,200 @@ protected static function get_mailpoet_extension_data() {
 		return $data;
 	}
 
+	/**
+	 * Get Creative Mail for WooCommerce extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_creative_mail_extension_data() {
+		$slug = 'creative-mail-by-constant-contact';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/creative-mail-by-constant-contact.png';
+
+		if ( 'activated' === $data['status'] && class_exists( '\CreativeMail\Helpers\OptionsHelper' ) ) {
+			if ( ! method_exists( '\CreativeMail\Helpers\OptionsHelper', 'get_instance_id' ) || \CreativeMail\Helpers\OptionsHelper::get_instance_id() !== null ) {
+				$data['status']      = 'configured';
+				$data['settingsUrl'] = admin_url( 'admin.php?page=creativemail_settings' );
+			} else {
+				$data['settingsUrl'] = admin_url( 'admin.php?page=creativemail' );
+			}
+
+			$data['docsUrl']    = 'https://app.creativemail.com/kb/help/WooCommerce';
+			$data['supportUrl'] = 'https://app.creativemail.com/kb/help/';
+		}
+
+		return $data;
+	}
+
+	/**
+	 * Get TikTok for WooCommerce extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_tiktok_extension_data() {
+		$slug = 'tiktok-for-business';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = WC_ADMIN_IMAGES_FOLDER_URL . '/marketing/tiktok.jpg';
+
+		if ( 'activated' === $data['status'] ) {
+			if ( false !== get_option( 'tt4b_access_token' ) ) {
+				$data['status'] = 'configured';
+			}
+
+			$data['settingsUrl'] = admin_url( 'admin.php?page=tiktok' );
+			$data['docsUrl']     = 'https://woocommerce.com/document/tiktok-for-woocommerce/';
+			$data['supportUrl']  = 'https://ads.tiktok.com/athena/user-feedback/?identify_key=6a1e079024806640c5e1e695d13db80949525168a052299b4970f9c99cb5ac78';
```

Comment:
There is a system error when I click on the link. Is this expected?

https://user-images.githubusercontent.com/4344253/187343722-fb6c5b66-ac39-4a7c-bb98-b86ed1eb05b9.mov